pkgs/development/python-modules/python-lsp-black/default.nix

```
@@ -10,15 +10,15 @@
 
 buildPythonPackage rec {
   pname = "python-lsp-black";
-  version = "1.3.0";
+  version = "2.0.0";
   format = "setuptools";
   disabled = pythonOlder "3.6";
```

```suggestion
  disabled = pythonOlder "3.8";
```
https://github.com/python-lsp/python-lsp-black/blob/744296948badb2ba8db4d5280555d614e095364a/setup.cfg#L25